package.json

```diff
@@ -16,7 +16,7 @@
     "@mui/lab": "^5.0.0-alpha.123",
     "@mui/material": "^5.11.13",
     "@mui/x-tree-view": "^7.7.0",
-    "@nismod/irv-api-client": "^0.8.0",
+    "@nismod/irv-api-client": "^0.9.0-dev.1",
```

Comment:
Once the `@nismod/irv-api-client` version `0.9.0` package is published, we should update this dependency as a final step. Right now, only the pre-released version is published in the package registry